operator/bundle/manifests/rhacs-operator.clusterserviceversion.yaml

```diff
@@ -894,6 +897,9 @@ spec:
   minKubeVersion: 1.15.0
   provider:
     name: Red Hat
+  relatedImages:
+  - image: ''
+    name: ''
```

Comment:
Hm, this bit looks weird. Can we avoid it easily?

Reply:
I think this was generated. Let's see if it's still there when I'm done with this.

Reply:
It's a generated file, not sure there's anything I can do about it.